content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+---
+title: "Blue Boot2Root Write-Up"
+date: 2020-11-11T13:43:28-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Blue host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+After participating in the Disney Can-You-Hack-It CTF, I've been trying to spend more of my time studying security and offensive penetration testing. Primarily, I've been trying to balance both understanding how to effectively perform an attack with the awesome frameworks out their like metasploit, but also gain a better understanding of what this framework is doing under the hood. This post is meant to be the first of many as I attempt to practice documenting both the attack and my methodology.
+
+## Environment
+Both attacks take place in a flat network consisting of my attack host, a freshly-booted Kali Linux livecd, and the the target host, a freshly booted Windows VM that I knew contained 3 flags to capture. No other information is known about the host, what it's running or it's OS versioning, however I would be lying if I said I didn't assume that this would be an eternalblue attack based off the name.
```

Comment:
> No other information is known about the host, what it's running or it's OS versioning, however I would be lying if I said I didn't assume that this would be an eternalblue attack based off the name.

No other information is known about the host, like what it's running or its OS version, but I would be lying if I said I didn't assume that this would be an eternalblue attack based off the name.